ui/lib/core/addon/components/replication-table-rows.js

```diff
@@ -2,6 +2,21 @@ import Component from '@ember/component';
 import { computed } from '@ember/object';
 import layout from '../templates/components/replication-table-rows';
 
+/**
+ * @module ReplicationTableRows
+ * The `ReplicationTableRows` component is table component.  It displays cluster mode details specific to the cluster of the Dashboard it is used on.
+ *
+ * @example
+ * ```js
+ * <ReplicationTableRows
+    @replicationDetails={{replicationDetails}}
+    @clusterMode="primary"
+    />
+ * ```
+ * @param {Object} replicationDetails=null - An Ember data object pulled from the Ember Model. It contains details specific to the whether the replication is dr or performance.
+ * @param {String} clusterMode=null - The cluster mode passed through to a table component. 
```

Comment:
it would be helpful to specify here whether this is 'dr' or 'performance' or 'primary' or 'secondary', since i know i always get them confused 😅

Reply:
Good call! Added.